website/addons/wiki/routes.py

```diff
@@ -120,6 +120,23 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/',
         ], 'delete', views.project_wiki_delete, json_renderer),
 
+        # Edit Permissions | PUT
+        Rule([
+            '/project/<pid>/wiki/permissions/<permissions>/',
```

Comment:
Ping @brianjgeiger for URL review.

I don't think the `<permissions>` variable belongs in the URL path. It probably belongs in the payload.